.gitignore

```diff
@@ -10,6 +10,7 @@ coverage/
 /tmp/
 rspec_logs.json
 test_output/
+.DS_STORE
```

Comment:
Should this be, or should we also have a line that ignores the `.DS_Store` (not all caps) spelling as well?